FTPFolderUploader/main.py

```diff
@@ -0,0 +1,293 @@
+                      dest="local_dir",
+                      help="[REQUIRED] Local Directory",
+                      default=None)
+    parser.add_option("-u", "--username",
+                      dest="username",
+                      help="[REQUIRED] username",
+                      default=None)
+    parser.add_option("-s","--server",
+                      dest="server",
+                      help="[REQUIRED] Server Address",
+                      default=None)
+    parser.add_option("-m",
+                      action="store_true",
+                      dest="monitor",
+                      help="Monitor directory changes SLEEP_SECONDS: %s" % SLEEP_SECONDS,
+                      default=None)
+    parser.add_option("-w",
+                      action="store_true",
+                      dest="walkdir",
+                      help="Walk sub directories of the given directory to find files to send.",
+                      default=None)
+
+    (options,args) = parser.parse_args()
+
+    if (options.username and options.server) or os.path.exists(default_config_file):
+
+        if os.path.exists(default_config_file):
+            sys.path.append('.')
+
+            config = configparser.ConfigParser()
+            config.read(default_config_file)
+
+            try:
+                server = config['ftp_conf']['server'] if options.server is None else options.server
+                username = config['ftp_conf']['username'] if options.username is None else options.username
+                local_dir = config['ftp_conf']['local_dir'] if options.local_dir is None else options.local_dir
+                monitor = eval(config['ftp_conf']['monitor']) if options.monitor is None else options.monitor
+                walkdir =  eval(config['ftp_conf']['walkdir']) if options.walkdir is None else options.walkdir
+
+                print('\n** Using local config file %s (and cmd overrides %s) **\n' % (default_config_file, options))
+
+            except KeyError as e:
+                print("ERROR --", str(e.args))
+                print()
+                print('Value(s) missing in %s file!  The following values MUST be included:' % default_config_file)
+                print('================================')
+                print('server = <server to ftp to>')
+                print('username = <Username for access to given server>')
+                print('local_dir = <local directory>')
+                print('monitor = True/False')
+                print('walkdir == True/False')
+                print('================================')
+                sys.exit()
+        else:
+            server = options.server
+            username = options.username
+            local_dir = options.local_dir
+            monitor = options.monitor
+            walkdir = options.walkdir
+
+            print('\n** Using: %s **\n' % options)
+
+        # get the user password
+        prompt = 'Password for (%s@%s) ?: ' % (username, server)
```

Comment:
This style of string formatting is discouraged these days, especially in Python 3 with f-strings. For example this could now be written as:
```python
prompt = f"Password for ({username}@{server}) ?: "
```